src/lighteval/tasks/tasks_prompt_formatting.py

```diff
@@ -14,6 +14,10 @@
 LETTER_INDICES = ["A", "B", "C", "D", "E", "F", "G", "H", "I", "J", "K", "L", "M", "N", "O", "P", "Q", "R", "S", "T", "U", "V", "W", "X", "Y", "Z"]
```

Comment:
please revert changes to this file and to tasks_table :)

Reply:
done